fenix/docs/Creating-a-release-branch.md

```diff
@@ -7,7 +7,22 @@ These are instructions for preparing a release branch for Fenix Beta release. Fo
 2. Verify that `version.txt` already refers to the Fenix Beta release version `[beta_version].0b1` in the `releases_v[beta_version].0.0` branch. Bump it manually if necessary.
 3. In the `main` branch, create a pull request to update `version.txt` to `[nightly_version].0b1`. Land the updated `version.txt` into `main` with a review from someone from RelMan (#releaseduty-mobile). See [#26177](https://github.com/mozilla-mobile/fenix/pull/26177) for example.
 4. Notify the dev team that they can start the new Nightly development cycle per the steps given in the next section ⬇️
-5. Once the new AC `v[beta_version].0b1` release is ready (see the [AC release checklist](https://mozac.org/contributing/release-checklist)), create a pull request targeting the `releases_v[beta_version].0.0` branch to pin the Android Components `VERSION` to `[beta_version].0b1`. The commit message should be `Set Android-Components version to [beta_version].0b1`.
+5. Update to the release version of application-services.  The commit message should be `Set application-services version to [release_version].0`.  (Note: application-services releases directly after the nightly cycle, there's no beta cycle).
+```diff
+diff --git a/android-components/plugins/dependencies/src/main/java/ApplicationServices.kt b/android-components/plugins/dependencies/src/main/java/ApplicationServices.kt
+index f64b26d7a1..72574ab057 100644
+--- a/android-components/plugins/dependencies/src/main/java/ApplicationServices.kt
++++ b/android-components/plugins/dependencies/src/main/java/ApplicationServices.kt
+@@ -3,8 +3,8 @@
+  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+-// These lines are generated by android-components/automation/application-services-nightly-bump.py
+-val VERSION = "113.20230320141535"
+-val CHANNEL = ApplicationServicesChannel.NIGHTLY_STAGING
++val VERSION = "113.0"
++val CHANNEL = ApplicationServicesChannel.RELEASE
+```
```

Comment:
I think this step makes sense, but we can be flexible here as well.  Whatever works for you is good with us.

I think that the Fenix beta release will need to depend on the app-services release.  We can coordinate and make sure our release goes out before you start your release process (maybe the work day before?).

Reply:
https://github.com/mozilla-mobile/firefox-android/blob/main/docs/contribute/release_checklist.md are the docs we should updating here.

Reply:
Right now, our practice for GeckoView is to do the s/NIGHTLY/BETA change immediately after creating the branch on merge day, then let relbot do the actual version bump once the first beta build is uploaded to Maven later on. I'm not sure if this approach is feasible here, though, since we'd be switching from XXX.YYYYMMDD format to XXX.0 also, but maybe?

But regardless, I think the right answer here with respect to timing is that the A-S team would do their branching and release tagging on merge day alongside the Gecko & firefox-android branching and then we can do the firefox-android bump afterwards.

Reply:
@rvandermeulen Looks like there's only this one question about what happens now with GV. ^ Seems like the last detail to address.

Reply:
For GeckoView, the Nightly builds upload to a different directory than the Beta & Release ones, so they don't end up living side-by-side in the same place.
https://maven.mozilla.org/?prefix=maven2/org/mozilla/geckoview/